### PR TITLE
Make crypt ta multi-instance

### PR DESCRIPTION
**Pull request by jenswi-linaro**
Signed-off-by: Jens Wiklander <jens.wiklander@linaro.org>

### CODE REVIEW

File: ta/crypt/include/user_ta_header_defines.h [hunk 1]
```diff
@@ -36,9 +36,7 @@
 	and TA_FLAG_MULTI_SESSION */
 /* It is not possible to open it twice in a raw */
 #define TA_FLAGS		(TA_FLAG_USER_MODE | TA_FLAG_EXEC_DDR | \
-				TA_FLAG_SINGLE_INSTANCE | \
-				TA_FLAG_MULTI_SESSION \
-				)
+				TA_FLAG_MULTI_SESSION )
```
Comment (jforissier) [inline ta/crypt/include/user_ta_header_defines.h]: I believe `TA_FLAG_MULTI_SESSION` has no effect when `TA_FLAG_SINGLE_INSTANCE` is not set. If so, you should remove it too.

Comment (jenswi-linaro) [inline ta/crypt/include/user_ta_header_defines.h]: You're right, I'll update.

**Comment by jenswi-linaro** [comment]
Update

**Comment by jforissier** [comment]
`Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Comment by jenswi-linaro** [comment]
Review tag applied

**Pull request merged**